jerry-core/config.h

```diff
@@ -159,6 +159,17 @@
 # define JERRY_CPOINTER_32_BIT 0
 #endif /* !defined (JERRY_CPOINTER_32_BIT) */
 
+/**
+ * Enable/Disable the engine's JavaScript debugger interface
+ *
+ * Allowed values:
+ *  0: Disable the debugger parts.
+ *  1: Enable the debugger.
+ */
+#ifndef JERRY_DEBUGGER
+# define JERRY_DEBUGGER 0
```

Comment:
After line 495 there are checks for the configuration macros. Could you please add it for this macro as well?